client/js/lounge.js

```diff
@@ -25,6 +25,7 @@ require("./clipboard");
 $(function() {
 	var sidebar = $("#sidebar, #footer");
 	var chat = $("#chat");
+	var searchQuery = $("#search-query");
```

Comment:
change this and the above two lines to `const`/`let`?

Reply:
Sorry but there is also others variables referring to elements below. I'm do not know all their use cases nor scopes. So I did like the others. See with the team what to do globally. My PR is maybe not the right place to do it.

Reply:
In new code I generally use let/const unless `var` is required.